src/content/api/assets.mdx

```diff
@@ -312,17 +249,6 @@ number.
 
 ---
 
-<Endpoint
```

Comment:
I would suggest keeping the list transactions endpoint in this file as it falls under the `/api/assets prefix`.